src/test/interpreters/interpreterService.node.ts

```diff
@@ -1,86 +0,0 @@
-/* eslint-disable implicit-arrow-linebreak */
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-import { inject, injectable } from 'inversify';
-import { Event, EventEmitter, Uri } from 'vscode';
-import { getInterpreterInfo } from './index.node';
-import { IPythonExtensionChecker } from '../../platform/api/types';
-import { Resource } from '../../platform/common/types';
-import { IInterpreterService } from '../../platform/interpreter/contracts';
-import { PythonEnvironment } from '../../platform/pythonEnvironments/info';
-import { ResourceSet } from '../../platform/vscode-path/map';
-
-let interpretersCache: Promise<PythonEnvironment[]> | undefined;
-@injectable()
-export class InterpreterService implements IInterpreterService {
```

Comment:
We never actually used this. At first I started making changes here but realized it wasn't even being used.